src/main/java/sc/iview/SciView.kt

```diff
@@ -1173,7 +1181,7 @@ class SciView : SceneryBase, CalibratedRealInterval<CalibratedAxis> {
     </T> */
     @Suppress("UNCHECKED_CAST")
     @Throws(Exception::class)
-    fun <T : RealType<T>?> addVolume(image: IterableInterval<T>): Node? {
+    fun <T : RealType<T>?> addVolume(image: IterableInterval<T>): Volume {
```

Comment:
For this and the next, why `RealType<T>?` and not `RealType<T>`...? Is this because of some Kotlin generics shenanigans? I know they can be a pain sometimes...

Reply:
I don't have an answer here as well, just kept it the way it was - will try what happens if I remove the `?`. It's a bit problematic that there are almost no tests in sciview. Would love to talk about how to improve that

Reply:
I removed the question mark.